tasks/blackduck-detect-task/ts/runner/DetectScriptConfigurationRunner.ts

```diff
@@ -13,7 +13,7 @@ import {IDefaultScriptConfiguration} from '../model/IDefaultScriptConfiguration'
 const osPlat: string = os.platform()
 
 export class DetectScriptConfigurationRunner extends DetectRunner {
-    static readonly DETECT_SH_SCRIPT_NAME = 'detect9.sh'
+    static readonly DETECT_SH_SCRIPT_NAME = 'detect9.sh' // TOME should point at 10 (SNAPSHOT for testing) along with sierra changes
```

Comment:
Check https://github.com/synopsys-sig/synopsys-detect-scripts/tree/gh-pages (TODO: update this to detect10-3.2.0-SNAPSHOT.ps1)